#!/bin/sh /usr/share/dpatch/dpatch-run
# Description: Make Xsession.d script ignore errors.
# Author: Michael Meskes <meskes@debian.org>

@DPATCH@
diff -urNad virtualbox-ose-3.0.8-dfsg~/src/VBox/Additions/x11/Installer/98vboxadd-xclient virtualbox-ose-3.0.8-dfsg/src/VBox/Additions/x11/Installer/98vboxadd-xclient
--- virtualbox-ose-3.0.8-dfsg~/src/VBox/Additions/x11/Installer/98vboxadd-xclient	2009-10-07 11:27:13.000000000 +0200
+++ virtualbox-ose-3.0.8-dfsg/src/VBox/Additions/x11/Installer/98vboxadd-xclient	2009-10-07 13:42:27.862988136 +0200
@@ -50,3 +50,5 @@
     /usr/bin/VBoxClient --display
 test -z "$no_display" &&
     /usr/bin/VBoxClient --seamless
+
+exit 0
